api/v1alpha1/edgedevice_types.go

```diff
@@ -64,10 +64,25 @@ type HardwareProfileConfiguration struct {
 
 // EdgeDeviceStatus defines the observed state of EdgeDevice
 type EdgeDeviceStatus struct {
-	Phase                     string      `json:"phase,omitempty"`
-	LastSeenTime              metav1.Time `json:"lastSeenTime,omitempty"`
-	LastSyncedResourceVersion string      `json:"lastSyncedResourceVersion,omitempty"`
-	Hardware                  *Hardware   `json:"hardware,omitempty"`
+	Phase                     string       `json:"phase,omitempty"`
```

Comment:
What are the expected value of the Phase attribute ?

Reply:
The statuses are sent by the device agent and for now it is only `up`